components/eam/src/physics/cam/modal_aer_opt.F90

```diff
@@ -409,78 +390,73 @@ end subroutine modal_aer_opt_init
 
 !===============================================================================
 
-subroutine modal_aero_sw(list_idx, dt, state, pbuf, nnite, idxnite, is_cmip6_volc, ext_cmip6_sw, trop_level,  &
-                         tauxar, wa, ga, fa)
-
+subroutine modal_aero_sw(dt, state, pbuf, nnite, idxnite, is_cmip6_volc, ext_cmip6_sw, trop_level,  & ! in
+                         tauxar, wa, ga, fa) ! out
    ! calculates aerosol sw radiative properties
 
-   integer,             intent(in) :: list_idx       ! index of the climate or a diagnostic list
-   real(r8),            intent(in) :: dt             !timestep (s)
-   type(physics_state), intent(in), target :: state          ! state variables
+  use mam_support, only : min_max_bound
 
+   real(r8),            intent(in) :: dt             !timestep [s]
+   type(physics_state), intent(in), target :: state          ! state variables
    type(physics_buffer_desc), pointer :: pbuf(:)
+
    integer,             intent(in) :: nnite          ! number of night columns
    integer,             intent(in) :: idxnite(nnite) ! local column indices of night columns
    integer,             intent(in) :: trop_level(pcols)!tropopause level for each column
    real(r8),            intent(in) :: ext_cmip6_sw(pcols,pver)
```

Comment:
You could put here a comment that this is short wave extinction with units of inverse meters.